lib/types/string.js

```diff
@@ -316,7 +316,23 @@ module.exports = Any.extend({
             alias: 'uuid',
             method(options = {}) {
 
-                Common.assertOptions(options, ['version', 'separator']);
+                Common.assertOptions(options, ['version', 'separator', 'wrapper']);
+
+                if (
+                    options.wrapper !== undefined &&
+                    options.wrapper !== true &&
+                    options.wrapper !== false
+                ) {
+                    const validWrappers = Object.keys(internals.guidBrackets).filter(
+                        (key) => !!key
+                    );
+                    assert(
+                        validWrappers.includes(options.wrapper),
+                        `"wrapper" must be true, false, or one of ${validWrappers.join(
+                            ', '
+                        )}`
+                    );
+                }
```

Comment:
I'd personally keep it simpler, this version feels more readable:
```suggestion
                assert(
                    options.wrapper === undefined ||
                    typeof options.wrapper === 'boolean' ||
                    (typeof options.wrapper === 'string' && typeof internals.guidBrackets[options.wrapper] === 'string'),
                    `"wrapper" must be true, false, or one of "${Object.keys(internals.guidBrackets).join('", "')}"`
                );

```